eng/MPack.props

```diff
@@ -1,5 +1,8 @@
 <Project>
   <PropertyGroup>
     <ArtifactsMPackDir>$(ArtifactsDir)MPack\$(Configuration)\</ArtifactsMPackDir>
+    <_MonoDevelopPath>$(MSBuildThisFileDirectory)..\..\monodevelop\</_MonoDevelopPath>
```

Comment:
This seems fragile and specific to your local environment. Suggest instead moving everything into the `RegisterAddin` target below _except_ setting `$(_MonoDevelopPath)`. Instead, use a `MonoDevelopPath` property that has no default, allowing users to specify `/p:MonoDevelopPath=..\..\monodevelop\` (or whatever matches their environment) on the command line.

Reply:
Just to clarify, VSmac development already requires these various repos to be cloned side-by-side, so it's not specific to David's setup. It is the required setup for all VSmac devs.

Reply:
:ok: I wasn't aware of that general requirement